data/settings/1.25.x/kubelet-device-plugin.toml

```diff
@@ -0,0 +1,152 @@
+[[docs.tag.nvidia]]
+heading = "Settings related to k8 nvidia device plugin"
+description = """
+See the [nvidia-k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin?tab=readme-ov-file#with-cuda-time-slicing) for more details.
+"""
+
+[[docs.tag.nvidia-time-slicing]]
```

Comment:
This should be its own section like `Bootstrap Containers`, and a simpler description should be provided in the API documentation.